packages/visualizations/src/components/utils/ColorsLegend.svelte

```diff
@@ -27,26 +61,50 @@
         </div>
     {:else if colorsScale.type === 'palette'}
         <!-- Palette color boxes, row display, no labels only displaying palettes steps -->
-        <div class="legend-colors-container-palette">
+        <div class="legend-colors-container-palette" bind:clientWidth={legendWidth}>
             <div class="legend-colors-row-color-box-palette">
                 {#each colorsScale.colors as color}
                     <div class="legend-colors-color-box-palette" style="--box-color: {color}" />
                 {/each}
             </div>
-            <div class="legend-colors-row-values-palette">
+            <div
+                class="legend-colors-row-values-palette {displayVertical
```

Comment:
Can't it be `class:vertical-labels-container="displayVertical"` like you did below? (I'm a Svelte noob)